unicore/ask/service/questions_api.py

```diff
@@ -76,9 +76,17 @@ def put(self):
             if value is not None and not attr == 'options':
                 setattr(question, attr, value)
 
-        for option in self.request.validated.get('options', []):
+        validated_options = self.request.validated.get('options', [])
+        # Delete existing options
+        if question.question_type != 'free_text':
+            existing_options = [
```

Comment:
What happens if a question is changed from `multiple_choice` to `free_text`?

Related question: do we want to delete options altogether, or maybe just delete the relation? I'm thinking of the case where there is existing answers for old options.